DOCSP-41733 query size limit (#817)
mongodb/docs-app-services: source/sync/error-handling/errors.txt

       <fs-indexed-queryable-fields-valid-client-side-queries>` for indexed 
       queryable fields.

   * - LimitsExceeded
     - This error indicates that the query being used has exceeded the size limit of 256
       kB.

>        To recover from this error, you may need to restructure your query to ensure the data
>        you are querying against falls within the acceptable limits. 

Reviewer: The wording "the data you are querying against" seems to imply that the data in your collection is less than 256 KB, and I don't believe that's what this error indicates. I believe it's the _size_ of the query you're using that is the issue. So I suggest we reword - maybe something like this?

Also - "you may need" isn't consistent with our style guide - we want to be authoritative in our recommendations and info. We can cut these as unnecessary wiggle words, and just present the solution.

```suggestion
       To recover from this error, restructure your query to ensure
       the data in your query falls within the acceptable size limit. 
```
